modules/primer-marketing-buttons/lib/button.scss

```diff
@@ -0,0 +1,30 @@
+.btn-orange { @include btn-solid($white, lighten($orange-500, 7.5%), darken($orange-500, 7.5%)); }
+.btn-blurple { @include btn-solid($white, mix($blue-400, $purple-400, 35%), mix($blue-600, $purple-600, 35%)); }
+
+.btn-outline-purple { @include btn-outline($purple); }
+.btn-outline-orange { @include btn-outline($orange); }
+.btn-outline-green { @include btn-outline($green); }
+
+@mixin btn-transparent-active {
+  color: #333;
```

Comment:
Forgot to add, can you replace the hex values with new color variables? We also have variables for line-height, border, and border-radius too (for use further down in `btn-transparent`.

Reply:
@broccolini will do! im actually going to remove a few declarations under`btn-transparent` on second thought a lot of these should be inherited from `.btn` already